src/components/obs/ObsHistory.jsx

```diff
@@ -73,8 +73,10 @@ class ObsHistory extends React.PureComponent {
 
   }
 
-  isEditableEncounter(encounter) {
-    return encounter && encounter.encounterType && this.props.editableEncounterTypes.map(e => e.uuid).includes(encounter.encounterType.uuid);
+  isEditableEncounter(encounter, obs) {
+    return encounter &&
+      (encounter.encounterType && this.props.editableEncounterTypes.map(e => e.uuid).includes(encounter.encounterType.uuid)) && // encounter type is in list of editable encounter types
```

Comment:
Do... uh... do we have style guidelines on long lines?

Actually, do we have style guidelines for JS/React?

Reply:
We use ESLint, but I'll have to refresh my memory on how it actually works....